test/functional/apps/discover/_field_data.js

```diff
@@ -25,7 +25,7 @@ export default function ({ getService, getPageObjects }) {
   const kibanaServer = getService('kibanaServer');
   const PageObjects = getPageObjects(['common', 'header', 'discover', 'visualize']);
 
-  describe('discover app', function describeIndexTests() {
+  describe('discover tab@', function describeIndexTests() {
     before(async function () {
```

Comment:
Why the `@` here?  I think that could cause problems with logging and with saving screenshots etc when there is a failure.